README.md

```diff
@@ -1,6 +1,6 @@
 # cpp-ethereum - Ethereum C++ client
 
-This repository contains [cpp-ethereum](http://cpp-ethereum.org), the [Ethereum](https://ethereum.org) C++ client.
+This repository contains [cpp-ethereum](http://cpp-ethereum.org), the [Ethereum](https://ethereum.org) C++ client, also known as Eth or Ceth.
```

Comment:
I have never heard about Ceth.

Reply:
No, it is more of a proposal to use Ceth to more readily distinguish it from ETH, the ticker for Ether, although I guess that in most contexts the two are readily distinguishable.